frontend/.env.sample

```diff
@@ -2,3 +2,4 @@
 // More information about env variables with Vite can be found here: https://vitejs.dev/guide/env-and-mode.html
 
 VITE_BASE_API_URL=http://localhost:8080
+VITE_BASE_CLIENT_URL=http://localhost:5173
```

Comment:
Please write a bit what BASE_CLIENT_URL is and how to use is (in frontend's README.md)

Reply:
Yes, about that.
I don't know why exactly this was introduced. I have to ask @badnames about that.

Maybe we can just remove the environment variable overall, as I think the default of '/' is fine in our setup.
Vite will append the variable as base path for static files it tries to serve.
https://vitejs.dev/guide/build.html#public-base-path

Reply:
Yes, if it is possible, removing the env var would be best!

@badnames can you maybe shine light on it?

Reply:
I will remove it bcs he added it by mistake.